Tests/Microsoft.AppCenter.Test.Functional/AppCenter/AppCenterCoreTest.cs

```diff
@@ -0,0 +1,203 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.AppCenter.Test.Functional.AppCenter
+{
+    using Analytics = Microsoft.AppCenter.Analytics.Analytics;
+    using AppCenter = Microsoft.AppCenter.AppCenter;
+
+    public class AppCenterCoreTest : IDisposable
+    {
+
+        // Before
+        public AppCenterCoreTest()
+        {
+            Utils.deleteDatabase();
+        }
+
+        // After
+        public void Dispose()
+        {
+            // Let pending SDK calls be completed.
+            Task.Delay(3000).Wait();
```

Comment:
Not needed if we wait to check on log payloads for start service logs or custom properties that happen after for that test.

Reply:
Will be fixed separately